dotnet-authserver/tests/TeacherIdentity.AuthServer.EndToEndTests/Register.cs

```diff
@@ -68,6 +71,93 @@ public async Task NewUserCanRegister()
             });
     }
 
+    [Fact]
+    public async Task NewUserWithTrnRequiredCanRegister()
```

Comment:
Ticket mentioned multiple e2e tests. I've added this one, is there something I'm missing?

Reply:
Would be good to cover existing email and existing mobile phone scenarios too.